addons/website_sale_stock/models/product_template.py

```diff
@@ -26,7 +26,9 @@ def _get_combination_info(self, combination=False, product_id=False, add_qty=1,
             return combination_info
 
         if combination_info['product_id']:
-            product = self.env['product.product'].sudo().browse(combination_info['product_id'])
+            website = self.env["website"].browse(self.env.context.get("website_id"))
+            company = website.company_id.id
```

Comment:
Is the company always assigned in the website or you should fallback to the company of the website user (`website.user_id.company_id`)?

Reply:
It's a required field.

Reply:
OK, thanks for confirming.